bootstrap/apollo-env.sh

```diff
@@ -14,3 +16,10 @@ if [[ -z "${color_start-}" ]]; then
   declare -r color_norm="${color_start}0m"
 fi
 
```

Comment:
How about we use the numeric as options, however only cater for 0 and N - then as we gain information to which log level combinations are useful create canned groups for them via APOLLO_LOG